include/kllvm/codegen/CreateTerm.h

```diff
@@ -1,13 +1,18 @@
 #ifndef CREATE_TERM_H
 #define CREATE_TERM_H
 
+#include <concepts>
+
 #include "kllvm/ast/AST.h"
 #include "kllvm/codegen/DecisionParser.h"
 
 #include "llvm/ADT/StringMap.h"
 #include "llvm/IR/Module.h"
 #include "llvm/IR/Value.h"
 
+// size up to which calls to the same allocation function will be merged.
+enum { MaxBlockMergeSize = 4096 };
```

Comment:
Perhaps it is a silly question, but is there any reason to use an enum instead of a global int var?

Reply:
Originally it was a macro, but clang tidy told me to use an enum, so I did. Not entirely sure the logic behind it. I probably could use a constexpr global too, if you think that's better. A regular global wouldn't necessarily get optimized as efficiently.